src/keyfile.c

```diff
@@ -745,9 +740,6 @@ static void load_dialog_prefs(GKeyFile *config)
 			g_key_file_set_boolean(config, "search", "pref_search_hide_find_dialog", suppress_search_dialogs);
 	}
 
-	/* read stash prefs */
-	settings_action(config, SETTING_READ);
```

Comment:
I had to move this down so the Stash group I add  later in this function gets loaded without additional calls.